lib/request_handler/validation/dry_engine.rb

```diff
@@ -34,7 +34,7 @@ def self.schema_class?(schema)
       end
 
       def self.validate!(value, schema, options: {})
-        validate(value, schema, options).tap do |result|
```

Comment:
isn't that behaving differently then before now?

Reply:
I believe that was a bug. But we don't have any method calling `validate!` with `options`, that is why we didn't see it. We can also drop forwarding `options` to `validate` since it is actually not used.